python/looker/rtl/user_session.py

```diff
@@ -19,6 +19,7 @@ class UserSessionError(Exception):
 class UserSession():
     """UserSession to provid automatic authentication
     """
+
```

Comment:
why doesn't my yapf do this? it's pep8 compliant which I think is the default style